git-some.sh

```diff
@@ -18,6 +18,12 @@ if grep --quiet --invert-match '^[[:digit:]]*$' <<< $commit_count; then
   exit 1
 fi
 
+COUNT=$(/bin/find . -maxdepth 1 -name "file-??.txt" | wc --lines)
+if [[ $(($COUNT + $commit_count)) -gt 100 ]]; then
+  echo "Can't create that many files!" >&2
+  exit 1
```

Comment:
Use 3 as the exit code (the script currently returns either 0, 1 or 2 depending on the error)